### ISSUE TITLE
Graph Export Issue with Nuitka Version

### ISSUE THREAD

**Issue by DJHostetler** [pyqtgraph/pyqtgraph#2276]
I am experiencing a persistent bug related to pyqtgraph and exporting data. I am also working with the nuitka support team to try and address this issue, but he has not been successful so far. I am able to successfully compile the program and run it with nuitka without errors. 

The error occurs after right clicking on the graph and selecting the Export button. If you have the Export window open and move the mouse in your graph area, then the program shuts down/exits without a known error.

### Code to reproduce
Included nuitka headers so that anyone can reproduce the same headings. After installing nuitka
```python
# nuitka-project: --standalone
# nuitka-project: --plugin-enable=pyside6
# nuitka-project: --include-qt-plugins=sensible,styles
# nuitka-project: --plugin-enable=numpy
# nuitka-project: --nofollow-import-to=PyQt5
# nuitka-project: --nofollow-import-to=PyQt6
# nuitka-project: --nofollow-import-to=PySide2
# nuitka-project: --include-package-data=pyqtgraph
# nuitka-project: --msvc=14.2
# nuitka-project: --python-flag=-v

from PySide6 import QtWidgets
from pyqtgraph import PlotWidget, plot
import pyqtgraph as pg
import sys
import os

class MainWindow(QtWidgets.QMainWindow):

    def __init__(self, *args, **kwargs):
        super(MainWindow, self).__init__(*args, **kwargs)

        self.graphWidget = pg.GraphicsLayoutWidget()
        self.setCentralWidget(self.graphWidget)

        hour = [1,2,3,4,5,6,7,8,9,10]
        temperature = [30,32,34,32,33,31,29,32,35,45]

        self.main_plot = self.graphWidget.addPlot(labels={'left': 'Hour', 'bottom': 'Temperature'}, labelTextSize='12pt')
        self.main_plot.plot(hour, temperature)
        self.graphWidget.setBackground('w')

if __name__ == '__main__':
    app = QtWidgets.QApplication(sys.argv)
    main = MainWindow()
    main.show()
    sys.exit(app.exec())
```

### Tested environment(s)

 * PySide version: 6.3.0
 * Qt Python binding: 6.3
 * Python version: 3.9
 * NumPy version: 1.22
 * Operating system: Windows (Has also been reproduced on Linux)
 * Installation method: pip

**Comment by j9ac9k** [comment]
Hi @DJHostetler,

PyQtGraph has some issues with PySide6 6.3; there are some upstream changes where pyqtgraph triggers segfaults right now.  I would advise first order of business to try PySide 6.2.x or PyQt6 6.3.

**Comment by DJHostetler** [comment]
Thanks @j9ac9k . I was having the same issue on PySide 6.1.3, but will try to see if 6.2.4 produces the same error.

**Comment by j9ac9k** [comment]
For the record, I don't think this issue will be solved by downgrading PySide versions to 6.2; but given 6.3 is failing our CI test suite for the time being, we are recommending people stay away from it for the time being.

**Comment by DJHostetler** [comment]
@j9ac9k, I can confirm that the error is reproduced for PySide 6.2.4.

**Comment by j9ac9k** [comment]
Hi @DJHostetler I can't reproduce this in a non-nuitka environment; how do I go about reproducing with nuitka (sorry I've never used it before) do I just run `python your_example.py` or is there some other way I need to call it?

**Comment by DJHostetler** [comment]
Nuitka can be installed via pip. 
[https://nuitka.net/doc/user-manual.html](url)
```python -m pip install nuitka```
You can download the code I posted above (which includes the necessary settings in the file header) and just run nuitka {path/to/python/file.py} in the command line. In the folder where you run the command, it will create an executable in a dist folder with supporting files.

**Comment by j9ac9k** [comment]
Yeah I got that far but I did get an error on the build step.  I put the console output above.

**Comment by DJHostetler** [comment]
@j9ac9k what OS are you using? I know nuitka runs on linux, but some variants may not be supported.

**Comment by j9ac9k** [comment]
This is on windows 11.

**Comment by DJHostetler** [comment]
Thanks @j9ac9k. Let me check with the nuitka maintainer to see if he has any ideas on why nuitka is throwing that error for Windows 11.

**Comment by j9ac9k** [comment]
> I can right click on the scene and bring up the window that does not include the export options without an error.

Oh that's interesting.  I'll take another look and see if something else looks suspicious.

**Comment by DJHostetler** [comment]
I just want to add that for now my solution is to disable the export window and add it as a separate option in another window. I have a workaround for my needs. 

I recognize the source of this issue could be coming from PySide, Qt, nuitka or pyqtgraph, which makes it very difficult to track. I started with the nuitka library because the issue shows up with a nuitka compiled example, but they were not successful in identifying the issue. If you can't get to the bottom of it, it's okay. I just wanted to raise the issue in case it is something that the library maintainer can easily identify.

**Comment by j9ac9k** [comment]
FWIW pyqtgraph is full of places where we work around upstream issues, so just because the fault is somewhere else doesn't mean we wouldn't accept a change in our code-base to make things work.  Before we can really help in this area, we would need to be able to reproduce the issue, and I recognize I may have nuitka not set up correctly (or using a development environment that may not be supported).

I'm good with leaving this issue open.
